utils/docker/Dockerfile-debbuild.ubuntu.18.04

```diff
@@ -36,8 +36,8 @@ ENV OFI_ART=\
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
             cmake libboost-dev libevent-dev pkg-config
 
-RUN curl -sS -f -L -O ${OFI_ART}/libfabric1_1.7.1%7Erc1-1_amd64.deb; \
-    curl -sS -f -L -O ${OFI_ART}/libfabric-dev_1.7.1%7Erc1-1_amd64.deb; \
+RUN curl -sS -f -L -O ${OFI_ART}/libfabric1_1.8.0-1_amd64.deb; \
+    curl -sS -f -L -O ${OFI_ART}/libfabric-dev_1.8.0-1_amd64.deb; \
```

Comment:
This is a good example of why fetching debs by URL like this is going to become untenable.  Rather than this package just picking up the latest of whatever is in the libfabric repo, when libfabric is updated like this, a flag-day event has to occur where every package that uses libfabric needs to be updated and pushed, etc.

I understand that we have some issues with creating debian repos at the moment, but this is a good example of why it will be important to resolve that ASAP.